src/WPML_OptionsManager.php

```diff
@@ -537,7 +537,23 @@ public function _LogMenu() {
                 $emailLogList->display();
                 ?>
             </form>
-            <?php $this->displayMP3Banner() ?>
+			<?php
+			$banner = $this->displayMP3Banner();
+            /**
+            * Control whether the banner is displayed or not
+            *
+            * @since 1.9.7
+            *
+            * @param boolean $display Whether the banner should be displayed. (default: true)
+            * @param string  $banner  The default banner content
+            * 
+            * @return boolean
+            */
+            $display = apply_filters('wpml_banner_display', true, $banner );
```

Comment:
Several points here:
1) There's an unnecessary space between `$banner` and `);`. Could you please remove it?
2) Is there a particular reason why `$banner` is being passed as an argument to the filter? Since there is only one controlled banner that we care about, I'd suggest to remove the `$banner` argument.